jQuery/src/index.js

```diff
@@ -0,0 +1,243 @@
+                            label: {
+                                template: labelTemplate("event"),
+                            },
+                        },
+                        {
+                            colSpan: 2,
+                            dataField: "Notes",
+                            editorType: "dxTextArea",
+                            editorOptions: {
+                                height: 90,
+                                maxLength: 200,
+                            },
+                            label: {
+                                template: (data, element) => {
+                                    const lineBreak = "<br>";
+                                    const infoIcon =
+                                        `<i id="helpedInfo" class="dx-icon dx-icon-info"></i>`;
+                                    const labelText = `Additional${lineBreak}${data.text}`;
+
+                                    element.append(
+                                        `<div id="template-content">${infoIcon}${labelText}</div>`
+                                    );
+
+                                    $("<div>").dxTooltip({
+                                        target: "#helpedInfo",
+                                        showEvent: "mouseenter",
+                                        hideEvent: "mouseleave",
+                                        contentTemplate(args) {
+                                            args.html(
+                                                `<div id="tooltip-content">This field must not exceed 200 characters</div>`
+                                            );
+                                        },
+                                    }).appendTo(element);
+                                },
+                            },
+                        },
+                        {
+                            dataField: "Phone",
+                            validationRules: validationRules.phone,
+                            editorOptions: {
+                                mask: "+1 (X00) 000-0000",
+                                maskRules: { X: /[02-9]/ },
+                                maskInvalidMessage:
+                                "The phone must have a correct USA phone format",
+                            },
+                            label: {
+                                template: labelTemplate("tel"),
+                            },
+                        },
+                        {
+                            dataField: "Email",
+                            validationRules: validationRules.email,
+                            label: {
+                                template: labelTemplate("email"),
+                            },
+                        },
+                    ],
+                }],
+            });
+    };
+
+    const popup = $("#popup").dxPopup({
+        hideOnOutsideClick: true,
+        height: 420,
```

Comment:
Use 'auto' here and remove height: "auto", from the 219 line

Reply:
Done